src/routes.json

```diff
@@ -246,5 +246,12 @@
         "routeAlias": "/microworlds",
         "view": "microworld/soccer/soccer",
         "title": "Soccer"
+    },
+    {
+        "name": "microworlds-homepage",
+        "pattern": "^/microworlds/go/?(\\?.*)?$",
+        "routeAlias": "microworlds/go/?\\??",
```

Comment:
This isn't explained anywhere, and we should fix that. But the `routeAlias` is used for shortening our URL regex for Fastly. You should reuse any `routeAlias` that already exists with the same prefix. Here, it should be `"/microworlds"`.